src/PrestaShopBundle/Resources/views/Admin/TwigTemplateForm/bootstrap_4_layout.html.twig

```diff
@@ -730,6 +730,26 @@
       {{ attributes.placeholder is defined ?  attributes.placeholder : 'Choose file(s)'|trans({}, 'Admin.Actions') }}
     </label>
   </div>
+  {# is defined in ImageTypeExtension #}
+  {% if image_url is defined and image_url is not null %}
+    <div class="form-group row mt-3">
+      <div class="col-sm">
+        <figure class="figure">
+          <img src="{{ image_url }}" class="figure-img img-fluid img-thumbnail">
+        </figure>
+        {% if delete_action_route is defined and id is defined %}
+        <div class="d-block">
```

Comment:
What ? Does this div has something else than a display block CSS sided? This class looks unnecessary

Reply:
It looks, but I was not able to make it other way this time. If you have a better suggestion, would be nice to hear it

Reply:
without it the button is aligned to the right of the other elements. With it, it is aligned below other elements of the block

Reply:
@NeOMakinG also this upload block has a strange behavior on it's left on the whole PrestaShop - all edit form using uploads. If I click it, it popups upload file dialog, even if I click not on the browse button. Could you check it out as a separate issue?

Reply:
> @NeOMakinG also this upload block has a strange behavior on it's left on the whole PrestaShop - all edit form using uploads. If I click it, it popups upload file dialog, even if I click not on the browse button. Could you check it out as a separate issue?

That behavior looks fine, it's same on basical html upload input : https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/file

Or I missed something of you explaination

Reply:
Do you have a simple way of testing your block so I can check the behavior of your div with that class ?

Did you test with a div without any classes?

Reply:
Without the div: 
![image](https://user-images.githubusercontent.com/32385991/76096538-f108aa00-5fce-11ea-8bbc-5d12c03cb1cf.png)

With the div:
![image](https://user-images.githubusercontent.com/32385991/76096586-02ea4d00-5fcf-11ea-93da-af3cc09e0010.png)

Issue with the click outside is that clicking at the form label area invokes upload popup. And if I extend the Upload block, the popup dialog opens even several centimeters from the upload input below the : Upload image file label.

Reply:
To easily test it you can try installing this module: https://github.com/PrestaShop/example-modules/pull/2 and go to Suppliers edit form

Reply:
Isn't that because the image file label has a margin maybe or something extending his height?

Reply:
The div is fine, but the class looks not, a div is naturally in display: block; that's why I commented : 
![image](https://user-images.githubusercontent.com/14963751/76097840-ab48e300-5fc8-11ea-8a47-b6bd5a3df294.png)